main.go

```diff
@@ -19,6 +19,9 @@ import (
 // Run the resource and datasource generation tool.
 //go:generate go run gen/generator.go
 
+// Format code and cleanup imports
+//go:generate go run -mod=mod golang.org/x/tools/cmd/goimports -w internal/provider/
```

Comment:
@akinross Looks like my next change to remove `-mod=mod` and use a different method to vendor the tool has been reverted.
Did you want me to add this back in or will you?